test/resource/igniter_test.exs

```diff
@@ -111,4 +111,99 @@ defmodule Ash.Resource.IgniterTest do
       |> assert_unchanged()
     end
   end
+
+  describe "add_new_code_interface" do
+    setup do
```

Comment:
Not sure if there's a simpler way to setup a Domain, Resource and Config sufficiently to exercise add_new_code_interface ?